src/Collision.java

```diff
@@ -0,0 +1,112 @@
+import edu.macalester.graphics.GraphicsGroup;
+import edu.macalester.graphics.Point;
+
+public class Collision {
+
+    private Snake snake;
+
+    private FoodManager foodPieces;
+
+    private WallManager wallManager;
+
+    private GraphicsGroup group;
+
+    
+    /**
+     * 
+     * Tests for various instances for collision throughout the game
+     * 
+     */
+    public Collision (Snake snake, FoodManager foodPieces, WallManager wallManager, GraphicsGroup group) {
+        this.foodPieces = foodPieces;
+        this.snake = snake;
+        this.wallManager = wallManager;
+        this.group = group;
+    }
+    
+    /**
+     * 
+     * Checks for collision between the snake and the food, removing the piece of food if true
+     * 
+     */
+
+    public boolean eatsFood() {
+        Point leftEye = snake.snakeEyes().get(0);
+        Point rightEye = snake.snakeEyes().get(1);
+
+        if (foodPieces.getElementAt(leftEye) == null && 
+                    foodPieces.getElementAt(rightEye) == null){
+            return false;
+        }
+        else if (foodPieces.getElementAt(leftEye) != null) {
+            foodPieces.remove(foodPieces.getElementAt(leftEye));
+            return true;
+        }
+        else {
+            foodPieces.remove(foodPieces.getElementAt(rightEye));
+            return true;
+        }
```

Comment:
You could even loop over `snakeEyes` and do away with all the many if branches; something like this:
```
for(Point eye : snake.snakeEyes()) {
  if (foodPieces.getElementAt(eye)) {
    return true;
  }
}
return false;